examples/rapids-image/.ipynb_checkpoints/app-image-config-input-checkpoint.json

```diff
@@ -0,0 +1,16 @@
+{
+    "AppImageConfigName": "custom-rapids-image-config",
+    "KernelGatewayImageConfig": {
```

Comment:
Same comment for this file re: `ipynb_checkpoints`.

Reply:
Please see above